tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -410,7 +410,9 @@ void test_inverse_map(const Map& map,
                       const std::array<T, Map::dim>& test_point) noexcept {
   INFO("Test inverse map");
   CAPTURE(test_point);
-  CHECK_ITERABLE_APPROX(test_point, map.inverse(map(test_point)).value());
+  const auto expected_test_point = map.inverse(map(test_point));
+  CHECK(expected_test_point.has_value());
```

Comment:
better to replace `CHECK` -> `REQUIRE` here (there's no point continuing the test when this check fails). Same for the test below